lightning-background-processor/src/lib.rs

```diff
@@ -75,10 +75,16 @@ const PING_TIMER: u64 = 1;
 /// Prune the network graph of stale entries hourly.
 const NETWORK_PRUNE_TIMER: u64 = 60 * 60;
 
-/// Trait which handles persisting a [`ChannelManager`] to disk.
+#[cfg(not(test))]
+const FIRST_NETWORK_PRUNE_TIMER: u64 = 60;
+#[cfg(test)]
+const FIRST_NETWORK_PRUNE_TIMER: u64 = 1;
+
+/// Trait which handles persisting a [`ChannelManager`] and [`NetworkGraph`] to disk.
 ///
 /// [`ChannelManager`]: lightning::ln::channelmanager::ChannelManager
-pub trait ChannelManagerPersister<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
+/// [`NetworkGraph`]: lightning::routing::network_graph::NetworkGraph 
```

Comment:
here and in a number of places you have EOL whitespace. A local `git show` should highlight them based on your terminal settings.